tests/test_on_data/test_gin_behavior.py

```diff
@@ -1,12 +1,55 @@
 import unittest
 from datetime import datetime
+from parameterized import parameterized, param
 
 from pynwb import NWBHDF5IO
-from neuroconv import NWBConverter, MovieInterface
+from neuroconv import NWBConverter, MovieInterface, DeepLabCutInterface
 
 from .setup_paths import OUTPUT_PATH, BEHAVIOR_DATA_PATH
 
 
+class TestDeepLabCutInterface(unittest.TestCase):
+    savedir = OUTPUT_PATH
+
+    @parameterized.expand(
+        [
+            param(
+                data_interface=DeepLabCutInterface,
+                interface_kwargs=dict(
+                    file_path=str(BEHAVIOR_DATA_PATH / "DLC" / "m3v1mp4DLC_resnet50_openfieldAug20shuffle1_30000.h5"),
+                    config_file_path=str(BEHAVIOR_DATA_PATH / "DLC" / "config.yaml"),
+                    subject_name="ind1",
+                ),
+            )
+        ]
+    )
+    def test_convert_behaviordata_to_nwb(self, data_interface, interface_kwargs):
+        nwbfile_path = self.savedir / f"{data_interface.__name__}.nwb"
+
+        class TestConverter(NWBConverter):
+            data_interface_classes = dict(TestBehavior=data_interface)
+
+        converter = TestConverter(source_data=dict(TestBehavior=dict(interface_kwargs)))
+        metadata = converter.get_metadata()
+        metadata["NWBFile"].update(session_start_time=datetime.now().astimezone())
+        converter.run_conversion(nwbfile_path=nwbfile_path, overwrite=True, metadata=metadata)
+
+        with NWBHDF5IO(path=nwbfile_path, mode="r", load_namespaces=True) as io:
+            nwbfile = io.read()
+            assert "behavior" in nwbfile.processing
+            processing_module_interfaces = nwbfile.processing["behavior"].data_interfaces
+            assert "PoseEstimation" in processing_module_interfaces
+
+            pose_estimation_series_in_nwb = processing_module_interfaces["PoseEstimation"].pose_estimation_series
+            expected_pose_estimation_series = ["ind1_leftear", "ind1_rightear", "ind1_snout", "ind1_tailbase"]
+
+            expected_pose_estimation_series_are_in_nwb_file = [
+                pose_estimation in pose_estimation_series_in_nwb for pose_estimation in expected_pose_estimation_series
+            ]
+
+            assert all(expected_pose_estimation_series_are_in_nwb_file)
```

Comment:
```suggestion
            assert all(
                x in processing_module_interfaces["PoseEstimation"].pose_estimation_series for x in ["ind1_leftear", "ind1_rightear", "ind1_snout", "ind1_tailbase"]
            )
```

Reply:
I really prefer the code that we have now. Two advantages of it:
1)
"assert (that) all(expected_pose_estimations_are_in_nwb_file)"
Reads to me more to me like natural language whereas the one liner has to be parsed to understand what the test is doing.
2) If the test fails and you throw a debugger before the assertion you can readily inspect which ones failed. To do this, you can just inspect the previously created variable `expected_pose_estimation_series_are_in_nwb_file` instead of having the re-do the one-liner again.

Reply:
Yeah, best to be as readable/modular as possible for test cases like this. For diagnosing it's a lot easier to introspect when they are pre-defined variables in the function prior to assertion fail

Reply:
My approach is a bit more performant by leveraging [Python's short-circuiting of logical operators](https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not), but I don't expect this to provide any real time improvement, and if you both feel the first way is easier to debug that's a good enough reason to keep it as-is.

Reply:
Yeah if this was actual source code I'd prefer it that way using the lazy generator approach.

Though also worth noting we can obtain even higher degree of test vebosity/introspection here by using this: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual

See https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/tests/test_inspector.py#L285 for example usage - the display is really nice when it fails (as long as maxDiff is set to None in the class)